lib/Data/Database.php

```diff
@@ -199,12 +199,31 @@ public function create($pasteid, array $paste)
             $burnafterreading = $paste['adata'][3];
         }
         try {
+            $big_string       = $isVersion1 ? $paste['data'] : Json::encode($paste);
```

Comment:
I don't exactly recall whether we have variable conventions, but I'd tend to use camelCasing for them.
Also, let's give it a proper name that describes what it contains, actually…

Something like:
```suggestion
            $pasteContent       = $isVersion1 ? $paste['data'] : Json::encode($paste);
```

Reply:
Or given I just see most (but not all – Uff I guess this needs some clreanup) variables use `underscore_casing`, then I'd suggest something like this:
```suggestion
            $paste_content       = $isVersion1 ? $paste['data'] : Json::encode($paste);
```